README.md

```diff
@@ -3,4 +3,47 @@
 [![CircleCI](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/develop.svg?style=svg)](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/develop)
 
 # Barefoot Nomad
+## Vision
+##### Making company travel and accomodation easy and convinient.
+
+## Description
+Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
+
+### Dependencies
+- Environment : [Nodejs](https://nodejs.org/)
+- Package Manager : [NPM](https://www.npmjs.com)
+- Compiler : [Babel](https://babeljs.io/)
+- Bundler : [Webpack](https://webpack.js.org/)
+- CSS Pre processor : [Sass](https://sass-lang.com)
+- Client Side : [Reactjs](https://reactjs.org),[Redux](https://redux.js.org)
+
+### Installation
+
+##### A. Clone the project
+1. From your computer, open terminal 
+2. Run `clone https://github.com/andela/team-odd-bn-frontend.git` OR [Download](https://github.com/andela/team-odd-bn-frontend/archive/develop.zip) the project
+
+##### B. Setting up the environment
+1. Create a file and name it `.env` in root directory
+2. Find a file named `.env.example`
+3. Copy & paste all environmental variables from `.env.example` to `.env`
+4. Provide values to all environmental variables in `.env` file
+
+### Run commands
+Open terminal from your computer
+1. Run `npm install` to install all dependencies
+2. Run `npm test` to test automatically if the is functioning properly as expected
+3. To start the app, run `npm start`. You will see the following details:
```

Comment:
To start development server run `npm run dev`.
To build app run `npm run build` and run `npm start`